src/components/ListTemplate/ColumnTemplates/DefaultColumn/DefaultColumn.js

```diff
@@ -0,0 +1,53 @@
+import React, { memo } from 'react';
+import HighLight from 'react-highlighter';
+import PropTypes from 'prop-types';
+
+import { AppIcon } from '@folio/stripes/core';
+
+import sharedCss from '../../../../shared.css';
+
+export const DefaultColumn = memo(props => {
+  const {
+    value,
+    className,
+    iconKey,
+    searchTerm,
+  } = props;
+
+  const recordContent = () => {
```

Comment:
Seems link that function can be replaced with a ternary operator as it is invoked in any return value.
```suggestion
  const content = searchTerm
    ? (
      <HighLight
        search={searchTerm}
        className={sharedCss.container}
      >
        {value}
      </HighLight>
    )
    : value;
```